gulpfile.js

```diff
@@ -471,7 +485,7 @@ const defaultTask = gulp.series(
     clean,
     assetsCopy,
     gulp.parallel(sassCompileAllFiles, lessCompileAllFiles),
-    gulp.parallel(jsCompileBundles, cssCompileBundles),
+    gulp.parallel(jsCompileBundles, cssCompileBundles, transpileJs),
```

Comment:
I think we'll likely need to transpile JS prior to bundling, so this should likely happen in parallel on the above line, when sass and less are also compiled?